### PR TITLE
Visit Table Row Clicking Event

### PR DESCRIPTION
**Pull request by Johngtka**
Features List:
- [X] `added selecting visit table row with row color changing`
- [X] `add storing select row id number to special storing point`
- [X] `to X button added closing method for confirmation dialog to unset patient visit`
- [x] `added option for table row when was second clicked then removed from storing point`

### CODE REVIEW

**Comment by Johngtka** [comment]
S&M commit proposal:
`feat: selecting visit row with storing id and change color of selected row (#34)`

File: src/app/visit/visit.component.ts [hunk 1]
```diff
@@ -54,8 +69,32 @@ export class VisitComponent implements OnInit {
     selectPatient(patientSelected: Patient): void {
         this.patient = patientSelected;
     }
+    clickedRow(row): void {
+        const index = this.visitPoints.indexOf(row.id);
+        if (index !== -1) {
+            this.visitPoints.splice(index, 1);
+            // console.log(this.visitPoints);
```
Comment (macrog) [inline src/app/visit/visit.component.ts:76]: remove commented out code

File: src/app/visit/visit.component.ts [hunk 2]
```diff
@@ -54,8 +69,32 @@ export class VisitComponent implements OnInit {
...
+        } else {
+            this.visitPoints.push(row.id);
+            // console.log(row.id);
```
Comment (macrog) [inline src/app/visit/visit.component.ts:79]: remove commented out code

**Comment by macrog** [comment]
**checklist:**

- [X] selecting a row, change color
- [X] deselection a row default color
- [X] store the selected row id in variable
- [X] visit with selected points, press _X_ button, shows warning dialog
  - [X] _cancel_ stays in current visit
  - [X] _confirm_ close the visit
- [ ] visit without selected point, press _X_ button, close the visit
     ⚠️ - warning dialog is shown ❗

**Review by macrog — APPROVED** [review]
works!

**Pull request merged**